optimizely/helpers/validator.py

```diff
@@ -151,3 +152,18 @@ def is_user_profile_valid(user_profile):
       return False
 
   return True
+
+
+def is_non_empty_string(input_id_key):
+  """ Determine if provided input_id_key is a non-empty string or not.
+
+  Args:
+    input_id_key: Variable which needs to be validated.
+
+  Returns:
+    Boolean depending upon whether input is valid or not.
+  """
+  if isinstance(input_id_key, string_types) and input_id_key:
```

Comment:
Perhaps flip it to be:

`if input_id_key and isinstance(input_id_key, string_types)`